main/main.cpp

```diff
@@ -44,9 +50,43 @@ static void initResources()
 
 int main(int argc, char** argv)
       {
+      // Force the 8-bit text encoding to UTF-8. This is needed on Windows, which would otherwise default to the local ANSI code page.
+      QTextCodec::setCodecForLocale(QTextCodec::codecForName("UTF-8"));
```

Comment:
this too may need to get done conditionally?

Reply:
I considered it, but I decided that it's a good thing to unconditionally force the encoding to UTF-8 on all platforms, since that's what we always want.

Reply:
But get anywhere else (anywhere but in MSVC) by default, right?

Reply:
So the comment at least should say that

Reply:
OK, will do.

Reply:
Done.